warp10/src/main/java/io/warp10/script/filter/FilterBySize.java

```diff
@@ -0,0 +1,96 @@
+//
+//   Copyright 2020  SenX S.A.S.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+package io.warp10.script.filter;
+
+import io.warp10.continuum.gts.GeoTimeSerie;
+import io.warp10.script.NamedWarpScriptFunction;
+import io.warp10.script.StackUtils;
+import io.warp10.script.WarpScriptException;
+import io.warp10.script.WarpScriptFilterFunction;
+import io.warp10.script.WarpScriptStack;
+import io.warp10.script.formatted.FormattedWarpScriptFunction;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class FilterBySize extends NamedWarpScriptFunction implements WarpScriptFilterFunction {
+    static private final String MIN = "min";
+    static private final String MAX = "max";
+
+    private final Long min;
```

Comment:
Make those primitive type. They can actually be int since the size of a GTS is an int, not a long.